### PR TITLE
Victor+Razvan=MidtermProject

### PR DESCRIPTION
**Pull request by zoldirazvan**
work in progress...

### CODE REVIEW

**Comment by VictorPop84** [comment]
We made a MVP homework, please check it out, thanks

File: midterm_project/midterm_project_victor_razvan/shopping.py [hunk 1]
```diff
@@ -0,0 +1,164 @@
+class PrittyPrinterAndMoreMixin:
```
Comment (Liitle) [inline midterm_project/midterm_project_victor_razvan/shopping.py:1]: Nice choice, but please find a more suitable name or split the two parts into two Mixins

**Review by Liitle — APPROVED** [review]
Good Job :smile:

**Comment by Liitle** [comment]
@VictorPop84  @zoldirazvan please change the base branch to be midterm_project

**Pull request merged**